test/xprof_vm_info_tests.erl

```diff
@@ -52,4 +52,67 @@ get_available_funs_test_() ->
               ?assertEqual([[?M, get_available_funs, 1]], L2),
               L3 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs/99">>),
               ?assertEqual([], L3)
-      end}].
+      end},
+     {"Match-spec fun matching",
+      fun() ->
+              L1 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs(_) ->">>),
+              ?assertEqual([[?M, get_available_funs, 1]], L1),
+              L1 = ?M:get_available_funs(<<"xprof_vm_info:get_available_funs return_trace()">>),
```

Comment:
`L1` is copy-paste error, should be `L2` indeed.

`return_trace()` itself is actually always included (that's how we measure time) so it makes no sense to include it in the match-spec (bad example in the test). But I would like to keep full power of match-specs if one wants to use it. And if we want to allow anything else than `/arity` there need to be some crazy cases. (Or maybe you need the dancing that is needed to avoid creating new atoms)

I am open for a checkbox if there is a frequently used match-spec feature fir which it makes sense.
(shortcut match-spec syntax with eg only args matching and no body is in the pipeline)